src/pages/Staking/HowItWorks/StakingApplications/index.tsx

```diff
@@ -0,0 +1,203 @@
+                  TBTC Node Docs
+                </Button>
+              </Link>
+            </VStack>
+          }
+          rewardSteps={[
+            "Run a tBTC node",
+            "Authorize a portion of your stake to tBTC",
+            "Have a staked balance",
+          ]}
+          aprPercentage={10}
+          slashingPercentage={1}
+        />
+        <ApplicationDetailsCard
+          preTitle="Random Beacon APP"
+          title="Random Beacon is a threshold relay that can generate verifiable randomness."
+          description="The Random Beacon application provides a trusted source of randomness for the process of trustless group election in the Threshold Network."
+          imgSrc={
+            colorMode === "dark"
+              ? randomBeaconAppIllustrationDark
+              : randomBeaconAppIllustrationLight
+          }
+          ctaButtons={
+            <VStack mb={6}>
+              <Link href="/somewhere" width="full">
+                <Button isFullWidth>Authorize Random Beacon</Button>
+              </Link>
+              <Link href="/somewhere" width="full">
+                <Button isFullWidth variant="outline">
+                  Random Beacon Node Docs
+                </Button>
+              </Link>
+            </VStack>
+          }
+          rewardSteps={[
+            "Run a Random Beacon node",
+            "Authorize a portion of your stake to Random Beacon",
+            "Have a staked balance",
+          ]}
+          aprPercentage={10}
+          slashingPercentage={1}
+        />
+        <ApplicationDetailsCard
+          preTitle="PRE APP"
+          title="Proxy Re-Encryption, or PRE, is cryptographic middleware for developing privacy-preserving applications."
+          description="PRE is a scalable end-to-end encryption protocol that allows a proxy entity to transform (or re-encrypt) encrypted data from one encryption key to another, without revealing the plaintext data. The nodes on the Threshold Network act as these proxy entities and use threshold cryptography to securely and cooperatively re-encrypt data for recipients based on access conditions defined by the data owner. "
+          imgSrc={
+            colorMode === "dark"
+              ? preAppIllustrationDark
+              : preAppIllustrationLight
+          }
+          ctaButtons={
+            <Link
+              href={ExternalHref.preNodeSetup}
+              target="_blank"
+              rel="noopener noreferrer"
+              mb={6}
+              width="full"
+            >
+              <Button isFullWidth variant="outline" mb={6}>
+                PRE Node Docs
+              </Button>
+            </Link>
+          }
+          rewardSteps={["Run a PRE node", "Have a staked balance"]}
+          aprPercentage={10}
+          slashingPercentage={1}
+        />
+      </Stack>
+    </Card>
+  )
+}
+
+StakingApplications.route = {
+  path: "applications",
+  index: false,
+  isPageEnabled: true,
```

Comment:
We should disable this page with feature flag

```suggestion
  isPageEnabled: featureFlags.MULTI_APP_STAKING,
```

Reply:
4e39e2de7f67e3ebc1c8dbccc2b0dac560cf325b